### ISSUE TITLE
Incorrect information synced if you switch user in a web browser

### ISSUE THREAD

**Issue by M1kep** [Taxel/PlexTraktSync#859]
### The problem

When browsing Plex in a Web Browser, if you utilize the "Switch User" functionality PlexTraktSync appears to makes calls to plex on behalf of the new user rather than the original user.

### Expected behavior

PlexTraktSync to only sync from a single home user profile

### Steps to reproduce the behavior

1. Start a sync. Is `plextraktsync`
2. While the sync is running, in a web browser Switch User to a different home user
3. PlexTraktSync will start syncing play statuses from the other users play history.

### Install method

Unraid

### Version

0.18.4

### Python Version

Whatever ships with the 0.18.4 Docker Image

### Operating System and Version

Unraid

### Config File ###
Default except for 1 additional excluded library

```
{
    "logging": {
        "debug": false,
        "filename": "plextraktsync.log",
        "append": true
    },
    "cache": {
        "path": "$PTS_CACHE_DIR/trakt_cache"
    },
    "sync": {
        "trakt_to_plex": {
            "watched_status": true,
            "ratings": true,
            "liked_lists": true,
            "watchlist": true
        },
        "plex_to_trakt": {
            "watched_status": true,
            "ratings": true,
            "collection": true
        }
    },
    "watch": {
        "scrobble_threshold": 80,
        "add_collection": false,
        "remove_collection": false,
        "username_filter": true
    },
    "xbmc-providers": {
        "movies": "imdb",
        "shows": "tvdb"
    },
    "excluded-libraries": [
        "Private",
        "Family Holidays",
	"Audiobooks"
    ]
}
```

**Comment by glensc** [comment]
> In a web browser Switch User to a different home user

what is that?

**Comment by twolaw** [comment]
This : [Fast User Switching in Plex](https://support.plex.tv/articles/204232453-fast-user-switching/)

@M1kep which user did you choose when you setup the PlexTraktSync script ? The main user or a managed user ?

**Comment by M1kep** [comment]
@twolaw I don't recall it prompting me, I just provided the script my username and password.

It almost seems like when I User Switch, the user my token runs as is also switched. Not sure if the script can detect this and throw an error?

**Comment by M1kep** [comment]
@twolaw Updated the OP. To also answer your questions here:

* Command: plextraktsync
* Switch user while the sync is still running
* Config added to OP

**Comment by M1kep** [comment]
I for some reason can't get this to reproduce, which is unfortunate as I'm positive something was funky. When I submitted this issue the sync was syncing two shows as "watched". When I checked the shows, I found that they were both marked as watched on the primary home users account(And not my user), who's user I was actively using in that moment. Seeing as I can't get this to actually reproduce I guess this can be closed.
